src/main/java/org/icgc/argo/program_service/controller/PublicProgramController.java

```diff
@@ -1,25 +1,34 @@
 package org.icgc.argo.program_service.controller;
 
 import java.util.List;
+import java.util.UUID;
+
+import lombok.val;
 import org.icgc.argo.program_service.model.dto.ProgramDTO;
+import org.icgc.argo.program_service.model.dto.GetJoinProgramInviteResponseDTO;
 import org.icgc.argo.program_service.model.dto.builder.ProgramDTOBuilder;
 import org.icgc.argo.program_service.model.entity.ProgramEntity;
 import org.icgc.argo.program_service.services.ProgramService;
+import org.icgc.argo.program_service.services.ProgramServiceFacade;
+import org.icgc.argo.program_service.converter.Grpc2JsonConverter;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
-@RequestMapping("/public")
+@RequestMapping("/public/programs")
```

Comment:
I don't think we wanted to change this, it will have impacts on the other endpoints in this controller... I might've missed this when reviewing the other PR.

Is it possible to move joinProgramInvite back to the other controller and just remove the auth restriction on it, so that we don't modify the paths?